db/migrate/20210123044336_add_tax_totals_to_order.rb

```diff
@@ -0,0 +1,14 @@
+class AddTaxTotalsToOrder < ActiveRecord::Migration
+  def up
+    add_column :spree_orders, :included_tax_total, :decimal,
+               precision: 10, scale: 2, null: false, default: 0.0
```

Comment:
In theory this should have a data migration to populate these fields on old records. The issue is that is might take a long time...

Reply:
It looks a bit like this, but for every order:
```ruby
additional_tax_total = order.all_adjustments.tax.additional.sum(:amount)
included_tax_total = order.line_item_adjustments.tax.sum(:included_tax) +
                     order.all_adjustments.enterprise_fee.sum(:included_tax) +
                     order.adjustments.shipping.sum(:included_tax)

order.update_columns(
  additional_tax_total: additional_tax_total,
  included_tax_total: included_tax_total
)
```

Reply:
That does look like it could take a while. Could we time it on a production db dump so we have a sense of whether we need to maybe break up the deployments so they go to each managed instance overnight? I'm about to pull AU for the other migration, I could test this one too.

Reply:
I recently added some changes to our cleanup script that should reduce the data lot, but it runs once per month via a cron job, and I don't think it's run on all instances yet... I was thinking we could trigger it manually via a rake task to make sure.